#
language: python
python: "3.6"
dist: xenial # trusty fails with weird FileNotFoundError in ExportContainerTask

sudo:  required
services:
    - docker

install:
  - sudo pip install pipenv

before_script:
  - pipenv install
  - export PYTHONPATH=$PYTHONPATH:$(pwd)

script:
  - pipenv run python3 build_utils/cli.py run-db-test --flavor-path build_utils/test/resources/test-flavor/ --release-type Release --workers 5

